### PR TITLE
Add tidb-backend necessary instructions

### PR DESCRIPTION
**Pull request by glkappe**
The document prompts are not clear, and the information currently provided does not allow tidb-lightning to run well

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migrate-from-mysql-dumpling-files.md [hunk 1]
```diff
@@ -45,6 +45,14 @@ aliases: ['/docs-cn/dev/migrate-from-mysql-mydumper-files/','/zh/tidb/dev/migrat
     password = ""
     ```
 
+3. 增加 tidb-backend 必要参数
+
+    ```
+    [tidb]
```
Comment (glorv) [inline migrate-from-mysql-dumpling-files.md:51]:
```suggestion
    [tikv-importer]
```

File: migrate-from-mysql-dumpling-files.md [hunk 2]
```diff
@@ -45,6 +45,14 @@ aliases: ['/docs-cn/dev/migrate-from-mysql-mydumper-files/','/zh/tidb/dev/migrat
...
+    # 由于当前模式使用 TiDb Backend 模式，可选参数有：“importer” 或 “local” 或 “tidb”
```
Comment (glorv) [inline migrate-from-mysql-dumpling-files.md:52]: 「TiDB Backend」。 另外，我们这里是不是应该描述让用户根据实际的场景选择使用的 Backend 并设置相关参数，并附加一下 Backends 页面的链接。其他两个 Backend, 还需要设置其他额外的配置项

File: migrate-from-mysql-dumpling-files.md (hunk 1 above)
Comment (glkappe) [reply migrate-from-mysql-dumpling-files.md]: done

File: migrate-from-mysql-dumpling-files.md (hunk 2 above)
Comment (glkappe) [reply migrate-from-mysql-dumpling-files.md:52]: 我增加了一些描述，辛苦帮忙 review，不足的地方请尽力指出，非常感谢，祝好~

File: migrate-from-mysql-dumpling-files.md [hunk 3]
```diff
@@ -45,6 +45,22 @@ aliases: ['/docs-cn/dev/migrate-from-mysql-mydumper-files/','/zh/tidb/dev/migrat
...
+    backend = "tidb"
+    ```
+    
+4. 增加导入 tidb 集群必要参数
+    
+    ```
+    [tidb]
+    status-port = {{tidb-status-port}}
+    pd-addr = {{pd-host}}:{{pd-port}}"
```
Comment (glorv) [inline migrate-from-mysql-dumpling-files.md:61]:
```suggestion
    host = "{{tidb-host}}"
    port = {{tidb-port}}
    user = "{{tidb-user}}"
    password = "{{tidb-password}}"
```

TiDB Backend 只需要设置 TiDB 连接相关的参数，pd 和 status-port 不需要设置

File: migrate-from-mysql-dumpling-files.md [hunk 4]
```diff
@@ -45,6 +45,22 @@ aliases: ['/docs-cn/dev/migrate-from-mysql-mydumper-files/','/zh/tidb/dev/migrat
     password = ""
     ```
 
+3. 增加 tidb-backend 必要参数
+
+    ```
+    [tikv-importer]
+    # 由于当前模式使用 TiDb Backend 模式，可选参数有：“importer” 或 “local” 或 “tidb”
```
Comment (glorv) [inline migrate-from-mysql-dumpling-files.md:52]:
```suggestion
    # 本教程采用 TiDb Backend 模式。此处也可以根据实际应用场景设置为 "local" 或 "importer". 具体请参考 [后端模式](/tidb-lightning-backends.md)
```

@TomShawn PTAL

Comment (TomShawn) [reply migrate-from-mysql-dumpling-files.md:52]:
```suggestion
    # 本教程采用 TiDB-backend 模式。此处也可以根据实际应用场景设置为 "local" 或 "importer"。具体请参考[后端模式](/tidb-lightning-backends.md)
```

**Comment by glorv** [comment]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**